admin-dev/themes/default/package.json

```diff
@@ -1,38 +1,39 @@
 {
-    "name": "default",
-    "version": "1.7.5",
-    "description": "",
-    "main": "index.js",
-    "scripts": {
-        "build": "webpack --bail --progress --mode production",
-        "watch": "webpack --bail --progress --watch --mode development"
-    },
-    "author": "PrestaShop",
-    "license": "OSL-3.0",
-    "devDependencies": {
-        "@babel/core": "^7.0.0-rc.1",
-        "@babel/preset-env": "^7.0.0-rc.1",
-        "@openfonts/ubuntu-condensed_latin": "^1.43.0",
-        "babel-loader": "^8.0.2",
-        "bourbon": "^4.3.4",
-        "clean-webpack-plugin": "^2.0.2",
-        "css-loader": "^1.0.0",
-        "file-loader": "^1.1.11",
-        "material-design-icons": "^2",
-        "mini-css-extract-plugin": "^0.4.2",
-        "node-sass": "^4.9.3",
-        "open-sans-fonts": "^1.5.1",
-        "optimize-css-assets-webpack-plugin": "^5.0.0",
-        "path": "^0.12.7",
-        "postcss-cssnext": "^3.1.0",
-        "postcss-loader": "^3.0.0",
-        "sass-loader": "^7.1.0",
-        "style-loader": "^0.22.1",
-        "uglify-save-license": "^0.4.1",
-        "webpack": "^4.17.0",
-        "webpack-cli": "^3.1.0"
-    },
-    "dependencies": {
-        "perfect-scrollbar": "^1.4.0"
-    }
+  "name": "default",
+  "version": "1.7.5",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --bail --progress --mode production",
+    "watch": "webpack --bail --progress --watch --mode development"
+  },
+  "author": "PrestaShop",
+  "license": "OSL-3.0",
+  "devDependencies": {
+    "@babel/core": "^7.0.0-rc.1",
+    "@babel/preset-env": "^7.0.0-rc.1",
+    "@openfonts/ubuntu-condensed_latin": "^1.43.0",
+    "babel-loader": "^8.0.2",
+    "bourbon": "^4.3.4",
+    "clean-webpack-plugin": "^2.0.2",
+    "css-loader": "^1.0.0",
+    "file-loader": "^1.1.11",
+    "material-design-icons": "^2",
+    "mini-css-extract-plugin": "^0.8.0",
+    "node-sass": "^4.13.1",
+    "open-sans-fonts": "^1.5.1",
+    "optimize-css-assets-webpack-plugin": "^5.0.0",
+    "path": "^0.12.7",
+    "postcss-cssnext": "^3.1.0",
+    "postcss-loader": "^3.0.0",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.3",
+    "uglify-save-license": "^0.4.1",
+    "uglifyjs-webpack-plugin": "^2.2.0",
```

Comment:
@NeOMakinG Have you checked the license for the package `uglifyjs-webpack-plugin` ?

Reply:
Was already used before but was centralized inside webpack, it's MIT

Reply:
:ok: